content/en/docs/faq/terminology.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "TLS Terminology"
+linkTitle: "TLS Terminology"
+weight: 50
+type: "docs"
+---
+
+With TLS being so widely deployed, terminology can sometimes get confused or be used to mean different things, and that reality
+combined with the complexity of TLS can lead to serious misunderstandings and confusion.
+
```

Comment:
Suggestion: we already linked it elsewhere, but maybe worth to link the TLS spec here too? https://datatracker.ietf.org/doc/html/rfc5246